source/Plugins/InstrumentationRuntime/MainThreadChecker/MainThreadCheckerRuntime.cpp

```diff
@@ -112,6 +112,7 @@ static std::string TranslateObjCNameToSwiftName(std::string className,
       swift::ClassDecl *cls = llvm::dyn_cast<swift::ClassDecl>(VD);
       if (!cls)
         return;
+      cls->loadAllMembers();
```

Comment:
This should not be necessary; `lookupDirect` should handle it. If that's not the case we should change something in Swift, not in MainThreadChecker.

Reply:
Yes, it is not the case. The lookup started returning wrong results after <https://github.com/apple/swift/pull/13866> landed. Do you have a sense of what's going on?

Reply:
Not obviously, but maybe there's a problem deserializing an AccessorDecl without its accompanying property? (See ModuleFile::loadObjCMethods.)